opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/wcaro/PNCVisit.java

```diff
@@ -0,0 +1,149 @@
+package org.smartregister.chw.activity.wcaro;
+
+import android.Manifest;
+
+import androidx.test.espresso.matcher.ViewMatchers;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+import org.smartregister.chw.R;
+import org.smartregister.chw.activity.LoginActivity;
+import org.smartregister.chw.activity.utils.Constants;
+import org.smartregister.chw.activity.utils.Utils;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static androidx.test.espresso.action.ViewActions.typeText;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withHint;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+
+public class PNCVisit {
+    @Rule
+    public ActivityTestRule<LoginActivity> mActivityTestRule = new ActivityTestRule<>(LoginActivity.class);
+
+    @Rule
+    public GrantPermissionRule permissionRule = GrantPermissionRule.grant(android.Manifest.permission.CALL_PHONE);
+
+    @Rule
+    public GrantPermissionRule mRuntimePermissionRule = GrantPermissionRule.grant(Manifest.permission.CAMERA);
+
+    @Rule
+    public GrantPermissionRule mRuntimePermissionRule1 = GrantPermissionRule.grant(Manifest.permission.ACCESS_FINE_LOCATION);
+
+    private Utils utils = new Utils();
+
+    public void setUp() throws InterruptedException {
+        Thread.sleep(1000);
+        utils.logIn(Constants.WcaroConfigUtils.wCaro_username, Constants.WcaroConfigUtils.wCaro_password);
+    }
+
+    @Test
+    public void confirmPNCVisit() throws Throwable {
+        utils.openDrawer();
+        onView(ViewMatchers.withSubstring(Constants.GenericConfigUtils.pnc))
+                .perform(click());
+        onView(withHint("Find name or ID"))
+                .perform(typeText("Ii"), closeSoftKeyboard());
+        onView(ViewMatchers.withSubstring("Ii Gg"))
+                .perform(click());
+        onView(withId(R.id.textview_record_visit))
+                .perform(click());
+        Thread.sleep(1000);
+        onView(ViewMatchers.withSubstring("Danger signs - mother"))
+                .check(matches(isDisplayed()));
+    }
+    @Test
+    public void successfullyRecordPNCVisit() throws Throwable {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)